change/@uifabric-azure-themes-819da265-3f92-400b-99db-6533f3df58ef.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Cherry picked from Azure theme 8.0 danger button commit",
```

Comment:
```suggestion
  "comment": "feat: Add styles for indeterminate checkbox and danger button.",
```

Reply:
I'll be sure to use more specific yarn change comments, thanks!